src/utils.js

```diff
@@ -1,6 +1,8 @@
 // @flow
-import { memoize, compact } from 'lodash'
+import { memoize, compact, pick, capitalize, every, find } from 'lodash'
```

Comment:
Yeah definitely not. I was hoping they wouldn't increase the bundle as much as they did :(